devices/common_patches/otg_fs_fixes.yaml

```diff
@@ -0,0 +1,96 @@
+OTG_FS_GLOBAL:
+  _add:
+    GRXSTSP_Device:
+      description: OTG status read and pop (device mode)
+      addressOffset: 0x20
+      size: 32
+      resetValue: 0x00000000
+      access: read-only
+      fields:
+        FRMNUM:
+          description: Frame number
+          bitOffset: 21
+          bitWidth: 4
+        PKTSTS:
+          description: Packet status
+          bitOffset: 17
+          bitWidth: 4
+        DPID:
+          description: Data PID
+          bitOffset: 15
+          bitWidth: 2
+        BCNT:
+          description: Byte count
+          bitOffset: 4
+          bitWidth: 11
+        EPNUM:
+          description: Endpoint number
+          bitOffset: 0
+          bitWidth: 4
+
+    GRXSTSP_Host:
+      description: OTG status read and pop (host mode)
+      addressOffset: 0x20
+      size: 32
+      resetValue: 0x00000000
+      access: read-only
+      alternateRegister: GRXSTSP_Device
+      fields:
+        PKTSTS:
+          description: Packet status
+          bitOffset: 17
+          bitWidth: 4
+        DPID:
+          description: Data PID
+          bitOffset: 15
+          bitWidth: 2
+        BCNT:
+          description: Byte count
+          bitOffset: 4
+          bitWidth: 11
+        CHNUM:
+          description: Channel number
+          bitOffset: 0
+          bitWidth: 4
+
+OTG_FS_DEVICE:
+  _array:
+    DIEPCTL[123]: {}
+    DOEPCTL[123]: {}
+
+  _modify:
+    FS_DIEPCTL0:
+      name: DIEPCTL0
+      displayName: DIEPCTL0
+
+    FS_DAINT:
+      name: DAINT
+      displayName: DAINT
+
+    FS_DAINTMSK:
+      name: DAINTMSK
+      displayName: DAINTMSK
+
+    FS_DIEPMSK:
+      name: DIEPMSK
+      displayName: DIEPMSK
+
+    FS_DOEPMSK:
+      name: DOEPMSK
+      displayName: DOEPMSK
+
+  DIEPCTL0:
+    _modify:
+      EPENA:
+        access: read-write
```

Comment:
This one is marked `r` in the RM so I'm not really sure what's going on with `EPENA`.

Reply:
The RM is wrong on this one (actually there are more access mistakes like that in the RM). It makes no sense for the bit to be `r`, because it would prevent from using the USB endpoint completely.

See this: <https://community.st.com/s/question/0D50X0000AAIG4fSQH/f429-usb-otg-in-endpoint-0-enable-bit-is-readonly-ref-manual-mistake>